samples/drivers/espi/prj.conf

```diff
@@ -1,4 +1,7 @@
 CONFIG_ESPI=y
 CONFIG_STDOUT_CONSOLE=y
 CONFIG_PRINTK=y
-
+# Uncomment to test SAF API's
+#CONFIG_ESPI_SAF=y
```

Comment:
please do not add commented options, create a new configuration with SAF enabled and configure it into sample.yaml so that it is built by CI.

Reply:
@scottwcpg Enable it only for EVB